q2_dada2/tests/test_denoise.py

```diff
@@ -24,6 +24,13 @@ def _sort_seqs(seqs):
     return sorted(list(seqs), key=lambda x: x.metadata['id'])
 
 
+class TestBase(TestPluginBase):
```

Comment:
Is TestBase the best name for this?

Reply:
Good question @ChrisKeefe - I pulled this from the formatting of our other unit tests (I've put usage examples in there before along with the testing set up) for future usability if any factories need to be initialized there, but I'm not attached to it - I could change it to TestExamples instead. Or add the test_examples method under the TestUtils() class?

Reply:
For posterity - we discussed offline and I have updated this class to TestExamples()